pom.xml

```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>com.spotify</groupId>
     <artifactId>foss-root</artifactId>
-    <version>5</version>
+    <version>9</version>
   </parent>
 
   <artifactId>styx</artifactId>
```

Comment:
Should we bump the major or middle(?) version too?

Reply:
👍

Reply:
https://semver.org/#what-should-i-do-if-i-update-my-own-dependencies-without-changing-the-public-api

Semver seems to suggest that we should bump either minor (middle) or patch.